examples/benchmark.py

```diff
@@ -0,0 +1,318 @@
+"""
+A script for testing / benchmarking HMM Implementations
+"""
+
+import argparse
+import collections
+import logging
+import time
+
+import hmmlearn.hmm
+
+import numpy as np
+
+import pandas as pd
```

Comment:
There's currently no dependency on pandas and I would rather keep things like that.

Reply:
Great Point, I will remove this dependency.